lasagne/tests/test_utils.py

```diff
@@ -1,8 +1,8 @@
-from mock import Mock
-import pytest
 import numpy as np
+import pytest
 import theano
 import theano.tensor as T
+from mock import Mock
```

Comment:
My bad, recently installed an automatic formatting tool on my editor and it's screwing things up :-( Will disable this